gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/BigQueryError.java

```diff
@@ -0,0 +1,118 @@
+package com.google.gcloud.bigquery;
+
+import com.google.api.services.bigquery.model.ErrorProto;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Google Cloud BigQuery error.
+ */
+public class BigQueryError implements Serializable {
```

Comment:
Is that used anywhere else other than Job? If not, would `JobError` be a better name?